.gitignore

```diff
@@ -1,23 +1,29 @@
-# Compiled class file
-*.class
+HELP.md
+!**/src/main/**/target/
+!**/src/test/**/target/
 
-# Log file
-*.log
+## Maven:
+!.mvn/wrapper/maven-wrapper.jar
+/.mvn/
+/mvnw
+/mvnw.bat
+mvnw.cmd
+/log/
+*.log*
+target/
+pom.xml.tag
+pom.xml.releaseBackup
+pom.xml.next
+release.properties
 
-# BlueJ files
-*.ctxt
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
 
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
+## Mac:
+/.DS_Store
```

Comment:
I still see DS_Store files in PR. They were versioned before. Please, remove them manually. Thank you!

Reply:
done

Reply:
Not everything were removed.

Reply:
Fixed